packages/shorebird_cli/lib/src/executables/gradlew.dart

```diff
@@ -75,15 +73,40 @@ class Gradlew {
     }
 
     final executablePath = executableFile.path;
-    final result = await process.run(
+    return process.run(
       executablePath,
-      ['app:tasks', '--all', '--console=auto'],
+      args,
       runInShell: true,
       workingDirectory: p.dirname(executablePath),
       environment: {
         if (!javaHome.isNullOrEmpty) 'JAVA_HOME': javaHome!,
       },
     );
+  }
+
+  /// Returns whether the gradle wrapper exists at [projectPath].
+  bool exists(String projectPath) {
+    final androidRoot = Directory(p.join(projectPath, 'android'));
+    return File(p.join(androidRoot.path, executable)).existsSync();
+  }
+
+  /// Return the version of the gradle wrapper at [projectPath].
+  Future<String> version(String projectPath) async {
+    final result = await _run(['--version'], projectPath);
+
+    final versionPattern = RegExp(r'Gradle (\d+\.\d+\.\d+)');
```

Comment:
This is pretty straightforward, but it's always a good idea with regexes to leave a comment demonstrating what they're matching.